src/convertToObject.js

```diff
@@ -12,7 +12,11 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const final = sourceString.split(';')
+    .map(rule => rule.split(':')
+      .map(rulePiece => rulePiece.trim()))
+    .filter(rule => rule.length > 1);
```

Comment:
```suggestion
  const final = sourceString.split(';')
    .map(rule => rule.split(':')
    .map(rulePiece => rulePiece.trim()))
    .filter(rule => rule.length > 1);
```
Why 'length > 1' but not > 0?

Reply:
The test doesn't work with 'length > 0' and linter is not happy with that change as well)

Reply:
Oh, I haven't noticed that you use the second 'map' INSIDE the first 'map', not AFTER it. Sorry. Ok. then the formatting is proper.

Reply:
Thank you for the review!